dist/init/linux-systemd/caddy.service

```diff
@@ -6,8 +6,11 @@ Wants=network-online.target systemd-networkd-wait-online.service
 
 [Service]
 Restart=on-failure
-StartLimitInterval=86400
-StartLimitBurst=5
+; Renamed to StartLimitIntervalSec for systemd>=230
+; Set StartLimitInterval=0 to disable rate limiting.
```

Comment:
It was not thought to be a recommendation, rather a hint for user that want to debug their caddy configuration.